backend.native/compiler/ir/backend.native/src/org/jetbrains/kotlin/backend/konan/llvm/IrToBitcode.kt

```diff
@@ -92,18 +92,29 @@ internal fun emitLLVM(context: Context) {
         devirtualizationAnalysisResult = Devirtualization.run(irModule, context, moduleDFG!!, externalModulesDFG!!)
 
         val privateFunctions = moduleDFG!!.symbolTable.getPrivateFunctionsTableForExport()
-
-        codegenVisitor.codegen.staticData.placeGlobalConstArray(irModule.descriptor.privateFunctionsTableSymbolName, int8TypePtr,
-                privateFunctions.map { constPointer(codegenVisitor.codegen.llvmFunction(it)).bitcast(int8TypePtr) },
-                isExported = true
-        )
+        privateFunctions.forEachIndexed { index, it ->
+            val function = codegenVisitor.codegen.llvmFunction(it)
+            LLVMAddAlias(
+                    context.llvmModule,
+                    function.type,
+                    function,
+                    irModule.descriptor.privateFunctionsTableSymbolName + "_$index"
```

Comment:
Maybe `"${irModule.descriptor.privateFunctionsTableSymbolName}_$index"`?

Reply:
Maybe. `privateFunctionsTableSymbolName` and `privateClassesTableSymbolName` must indeed be refactored, but the purpose of this PR is to fix regression as fast as possible.

Reply:
Done.